frontend/src/components/main-page/groups-tree-page/group-details/group-details.tsx

```diff
@@ -14,7 +28,72 @@ export const GroupDetails: React.FC = () => {
 
   return (
     <React.Fragment>
-      Group Details {JSON.stringify(controller.group)}
+      {controller.group && (
+        <Box
+          sx={{
+            overflowX: 'hidden',
+            overflowY: 'auto',
+            padding: 4,
+            maxWidth: '700px',
+          }}
+        >
+          <Typography variant="h3">Group Information</Typography>
+
+          {Object.keys(controller.group).includes('name') && (
+            <List>
+              <ListItem divider>
+                <ListItemIcon>
+                  <Badge />
```

Comment:
`Badge` somehow not displayed.

Reply:
That's because of a wrong import. You imported `Badge` from `@mui/material`, but you probably wanted to import it from `@mui/icons-material`.

This has happened multiple times to me. IMHO, it would be much better for DX to have a suffix like `Icon`, so that you have to import `BadgeIcon` instead of just `Badge`. I have a (pretty hacky) idea that we could use for this. Let's talk about that in the next meeting. 😃

Reply:
Thx :)